helm/botkube/e2e-test-values.yaml

```diff
@@ -180,7 +180,7 @@ executors:
         resources: [ "deployments" ]
 
   'plugin-based':
-    botkube/echo:
+    botkube/echo@v0.1.0:
```

Comment:
To cover detected bug, where plugin was not resolved when the exact version is specified